Compiler-Internals.md

```diff
@@ -18,19 +19,27 @@ The checker then waits until a particular symbol is requested to calculate type
 Each phase of the compiler (parser, binder, etc -- see below for details) treats data structures from the previous phases as immutable.
 In addition, data structures created within each phase are not usually modified after their creation.
 This requires a look-aside table in some cases.
-For example, because the binder only looks at one file at a time, the checker needs a merged-symbols table to track merged declarations. 
+For example, because the binder only looks at one file at a time, the checker needs a merged-symbols table to track merged declarations.
 It checks whether a symbol has an entry in the merged-symbols table each time before it uses a symbol.
 
 # Parser
 
+It's a recursive descent parser. It's pretty resilient, so if you search for functions matching the thing you want to change, you can probably get away with just adding the code to parse.
```

Comment:
Sentences on each line

I also don't understand the second sentence and the causality from being pretty resilient.